robowflex_library/src/trajectory.cpp

```diff
@@ -169,6 +169,12 @@ std::vector<std::string> Trajectory::getJointNames() const
     return getMessage().joint_trajectory.joint_names;
 }
 
+Trajectory &Trajectory::append(const Trajectory &source, double dt, size_t start_index, size_t end_index)
+{
+    trajectory_->append(*source.getTrajectoryConst(), dt, start_index, end_index);
+    return *this;
```

Comment:
I don't think you need to return the trajectory here. making it a void function should cover all  use cases as well, since this does affect the underlying trajectory.

Reply:
That's true, I'm returning the reference just to follow the convention from [MoveIt!'s trajectory append](http://docs.ros.org/en/noetic/api/moveit_core/html/cpp/classrobot__trajectory_1_1RobotTrajectory.html#a9f13f4305a66b59b8f0d4c30e26125a8). Returning the reference just lets it be passed if necessary immediately.